helm/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: eks-chart
 description: A Helm chart for the ACK service controller for Amazon Elastic Kubernetes Service (EKS)
-version: v1.0.0
-appVersion: v1.0.0
+version: v1.0.1
+appVersion: v1.0.1
```

Comment:
Same comment as above. The only way this would have changed is if you had a `RELEASE_VERSION` environment variable set with a value of `v1.0.1`, and the only reason you'd have that would be if you were going through the release process and creating a release PR...